packages/select/src/Select/Select.tsx

```diff
@@ -31,7 +31,10 @@ export const getSelectChildren: CollectionChildren<SingleItemType> = item => (
   <Item key={item.value}>{item.label}</Item>
 )
 export interface SelectProps
-  extends OverrideClassName<Omit<AriaSelectProps<SingleItemType>, "children">> {
+  extends OverrideClassName<
+    Omit<AriaSelectProps<SingleItemType>, "children" | "disabledKeys">
+  > {
+  disabledKeys?: React.Key[]
```

Comment:
I re-instantiated the disabledKeys prop as the type from React Aria was Iterable<T> - I felt it would be easier for the consumer to pass in an array of Keys

Reply:
How are consumers currently disabling their select options?

Reply:
In filterMultiSelect...
<img width="593" alt="image" src="https://user-images.githubusercontent.com/48232362/207777533-750fc6ab-ae94-4cc5-91d5-e9c66976988c.png">

Reply:
(we dont have any consumers actually using the Select lol)

Reply:
They are using the old one aren't they?

Reply:
<img width="665" alt="image" src="https://user-images.githubusercontent.com/48232362/207778245-2c753a8c-b366-4496-8e96-f5499cbb356f.png">

Reply:
Gets passed through the items 🤮

Reply:
Hmm this is a tough one, but I guess no matter the format keys/items you can still apply more or less the same methods to determine what are disabled and use that in your keys.

Reply:
One thing though, the term "keys" is confusing, cause when defining the items, none of them are keys, if it maps to value, I'd go with `disabledValues`

Reply:
I updated to `disabledValues`